packages/remediations/src/RemediationButton.js

```diff
@@ -57,7 +57,7 @@ RemediationButton.propTypes = {
 RemediationButton.defaultProps = {
     isDisabled: false,
     onRemediationCreated: f => f,
-    children: 'Remediate with Ansible'
+    children: 'Plan remediation'
```

Comment:
The mocks modify terminology but that is not in effect currently. The old terminology should be used as described in https://marvelapp.com/j3b9b7e/screen/63083483
I.e., this title should not change. Sorry for confusion.